tests/test_jexl.py

```diff
@@ -1,5 +1,8 @@
 import pytest
 
+import hypothesis
+from hypothesis import strategies
```

Comment:
Why import both? Why not just do `JEXL_ALPHABET = hypothesis.strategies.characters(...` instead?